model_compression_toolkit/quantizers_infrastructure/pytorch/inferable_quantizers/activation_inferable_quantizers/activation_uniform_inferable_quantizer.py

```diff
@@ -12,15 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+import numpy as np
 
 from model_compression_toolkit.core.common.constants import FOUND_TORCH
 from model_compression_toolkit.quantizers_infrastructure.common.base_inferable_quantizer import QuantizationTarget
+from model_compression_toolkit.quantizers_infrastructure.pytorch.inferable_quantizers \
```

Comment:
doesn't importing from a pytorch folder requires it to be located under FOUND_TORCH?